### ISSUE TITLE
Styling of text (overlay) does not appear as per demo (with Polymer cli starter kit)

### ISSUE THREAD

**Issue by gdevacc12** [mpachnis/mp-slider#2]
The styling of the text overlay does not appear correct (see attached image) when used with Polymer cli starter kit.
See http://test.hebrideanseatours.co.uk for an example

Same result with a range of browsers:
Windows browsers:
Firefox 53.0
Opera 44.0.2510.1449
Chrome 58.0.3029.81 (64-bit)
Edge 38.14393.0 (EdgeHTML 14.14393)
Internet Explorer 11.953.14339.0
Android:
Chrome 58.0.3029.83
Opera 42.7.2246.114996
Linux browsers:
Opera 44.0.2510.1449
Firebox 53.0
Chrome Version 58.0.3029.81 (64-bit)
<img width="487" alt="image" src="https://cloud.githubusercontent.com/assets/1928736/25614165/46262660-2f4a-11e7-8441-b462e86f57b1.png">

**Comment by mpachnis** [comment]
The caption is now separated. You should change you code like this:
```html
<mp-slider controls bullets show-thumbs>
    <div>
        <img src="image_path" />
        <mp-caption slider-header="Caption Header"
                    slider-content="Caption paragraph/text">
        </mp-caption>
    </div>
</mp-slider>
```

**Comment by gdevacc12** [comment]
That works thanks but the slider-content (see attached image from http://test.hebrideanseatours.co.uk) appears too high. Perhaps should be vertical middle in the slider-content?
(Note: Documentation  appears out of step at: http://minas.pachnis.com/projects/mp-slider/bower_components/mp-slider/)
![image](https://cloud.githubusercontent.com/assets/1928736/25648927/53f6b5de-2fea-11e7-83e6-28c6ce68ff93.png)

**Comment by mpachnis** [comment]
Did you change the mp-slider.html file? There is a nested `mp-caption` tag in your file. Check this [line](https://github.com/mpachnis/mp-slider/blob/master/mp-slider.html#L43), in your code looks like this:

```html
<template is="dom-if" if="{{caption}}">
                    <mp-caption><mp-caption>
                </mp-caption></mp-caption></template>
```

I just create a new starter-kit, install the element with bower and seems ok, there is no duplicate `mp-caption` tag.

I'll update the doc, thanks.

**Comment by gdevacc12** [comment]
No changes were made to mp-slider.html. To check I uninstalled and re-installed mp-slider.
It looks like:
https://github.com/mpachnis/mp-slider/blob/master/mp-slider.html contains (lines: 42,43,44): 
\<template is="dom-if" if="{{caption}}">
      \<mp-caption><mp-caption>
\</template>"
I discovered the problem.
Previously I missed removing "caption" from <mp-slider ...> (it was used in the previous version demo but I missed that it was removed in your example for this version's demo).
Once I made the code as per your example (separate mp-caption and removing caption parameter) then all appears to work fine. 
Perhaps the above template definition is now redundant in mp-slider.html v0.1.2 and can be removed?
Also the example in the mp-slider.html comments needs updating to this version.
Many thanks for sharing and help in diagnosis.
